packages/contentstack-export/src/commands/cm/stacks/export.ts

```diff
@@ -117,8 +117,7 @@ export default class ExportCommand extends Command {
       log({ data: exportDir } as ExportConfig, `Failed to export stack content - ${formatError(error)}`, 'error');
       log(
         { data: exportDir } as ExportConfig,
-        `The log has been stored at ${
-          exportDir ? path.join(exportDir, 'logs', 'export') : path.join(__dirname, 'logs')
```

Comment:
@cs-raj , we created an export folder inside the log directory. Make sure it functions in the same manner, please.

Reply:
Checked it it's working fine